superset-frontend/src/dashboard/components/Header/index.jsx

```diff
@@ -496,6 +496,7 @@ class Header extends PureComponent {
     const refreshWarning =
       dashboardInfo.common?.conf
         ?.SUPERSET_DASHBOARD_PERIODICAL_REFRESH_WARNING_MESSAGE;
+    const isEmbedded = !dashboardInfo?.userId;
```

Comment:
[non blocking] curious whether there's a more explicit way to check whether we're in an embedded context?

Reply:
I don't think so, but this comes up often... we need to invent this kind of state/check for myriad reasons.

Reply:
I agree it's not ideal, but this is consistent with other places we assess if it's an embedded user or not.